api/WindowClient.json

```diff
@@ -59,7 +59,7 @@
               "version_added": null
             },
             "edge": {
-              "version_added": null
+              "version_added": false
```

Comment:
Does any browser support `ancestorOrigins` or should this entry be removed entirely?

Reply:
It doesn't seem like any browsers support this, and there's no MDN web docs page for it.  Removing this entry sounds like the best way to go about it!

Reply:
Do you want to send a removal PR instead then?

Reply:
It's already been done and merged, actually!  Florian reviewed and approved #10415.

Reply:
Oh wait, that wasn't the only change :)